metaflow/plugins/kubernetes/kubernetes_job.py

```diff
@@ -53,12 +56,432 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def _make_domain_name(
+    jobset_name, main_job_name, main_job_index, main_pod_index, namespace
+):
+    return "%s-%s-%s-%s.%s.%s.svc.cluster.local" % (
+        jobset_name,
+        main_job_name,
+        main_job_index,
+        main_pod_index,
+        jobset_name,
+        namespace,
+    )
+
+
+class KubernetesJobSet(object):
+    def __init__(self, client, **kwargs):
+        self._client = client
+        self._kwargs = kwargs
+        self._group = "jobset.x-k8s.io"
+        self._version = "v1alpha2"
+        self.name = kwargs.get("name")
+
+        use_tmpfs = self._kwargs["use_tmpfs"]
+        tmpfs_size = self._kwargs["tmpfs_size"]
+        tmpfs_enabled = use_tmpfs or (tmpfs_size and not use_tmpfs)
```

Comment:
are these actually supported today? can you add a note in the comments - what functionality do we lack compared to vanilla @kubernetes?